ethexe/service/src/lib.rs

```diff
@@ -262,6 +271,7 @@ impl Service {
         router_query: RouterQuery,
         processor: ethexe_processor::Processor,
         signer: ethexe_signer::Signer,
+        tx_pool: TxPoolService,
```

Comment:
pretty much interesting question should tx pool be an optional service: basically, it depends both on network and rpc which are optional, but when they're enabled, it must be enabled as well

Reply:
as discussed added TODO for the network